### PR TITLE
Add the ability to ignore specific tests with IGNORES and have AppVeyor do so

### PR DESCRIPTION
**Pull request by JLockerman**
`bgw_db_scheduler` is flaky enough that it's not worth running on AppVeyor right now

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -138,7 +138,7 @@ test_script:
 
     #right now we only run timescale regression tests, others will be set up later
 
-    docker exec -e TEST_TABLESPACE1_PATH="C:\Users\$env:UserName\Documents\tablespace1\" -e TEST_TABLESPACE2_PATH="C:\Users\$env:UserName\Documents\tablespace2\" -e TEST_SPINWAIT_ITERS=10000 -e USER=postgres -it pgregress /bin/bash -c "cd /timescaledb/build && make regresschecklocal"
+    docker exec -e IGNORES="bgw_db_scheduler" -e TEST_TABLESPACE1_PATH="C:\Users\$env:UserName\Documents\tablespace1\" -e TEST_TABLESPACE2_PATH="C:\Users\$env:UserName\Documents\tablespace2\" -e TEST_SPINWAIT_ITERS=10000 -e USER=postgres -e -it pgregress /bin/bash -c "cd /timescaledb/build && make regresschecklocal"
```
Comment (svenklemm) [inline appveyor.yml]: seems like the -e after USER=postgres is superfluous

File: appveyor.yml [hunk 2]
```diff
@@ -138,7 +138,7 @@ test_script:
...
```
Comment (RobAtticus) [inline appveyor.yml]: Looks like the last `-e` is a stray?

Comment (JLockerman) [reply appveyor.yml]: indeed

File: appveyor.yml (hunk 1 above)
Comment (JLockerman) [reply appveyor.yml]: indeed

**Pull request merged**